hazelcast-client-legacy/src/test/java/com/hazelcast/client/topic/ClientReliableTopicTest.java

```diff
@@ -67,13 +79,13 @@ public void testCreation() {
 
     @Test(expected = NullPointerException.class)
     public void addMessageListener_whenNull() {
-        ITopic topic = client.getReliableTopic("foo");
+        ITopic topic = client.getReliableTopic(randomString());
```

Comment:
What is the added value of randomization of the topic name? 

Marko suggested removing randomization from tests:
https://hazelcast.atlassian.net/wiki/display/PM/Remove+Randomness+from+Unit+Tests